Update UG and fix Autocomplete (#212)
AY2223S2-CS2103T-T11-2/tp: docs/UserGuide.md

**:information_source: Note:**<br>

* Autocomplete only works for command words and prefixes that expect a corresponding input data. You may refer to the

> [Prefix](#prefix) section for more details on prefixes.<br>

Reviewer: do we need the `<br>`? wouldnt point form (`*`) already break lines automatically?

Author: Yah true, think I forgot to remove it. Removed

> * If input matches a command, Autocomplete will autocomplete the [prefixes](#prefix) for the command. If command requires no prefix, autocomplete will not do anything else.

Reviewer: ```suggestion
* If input matches a command, Autocomplete will autocomplete the [prefixes](#prefix) for the command. If the command does not require any prefix, autocomplete will not do anything.
```
